x-pack/plugins/watcher/public/application/models/watch/threshold_watch.js

```diff
@@ -261,6 +261,7 @@ export class ThresholdWatch extends BaseWatch {
       timeWindowSize: this.timeWindowSize,
       timeWindowUnit: this.timeWindowUnit,
       hasTermsAgg: this.hasTermsAgg,
+      isActive: this.watchStatus.isActive,
```

Comment:
I think we need to make this change to the `base_watch` model instead: https://github.com/elastic/kibana/blob/master/x-pack/plugins/watcher/public/application/models/watch/base_watch.js#L117

As it stands, this change will only affect threshold watches. If we make the change to the base watch model, it will affect the advanced/JSON watches too. I tested locally and verified that advanced/JSON watches still exhibit the bug with the current changes, but moving this change to `base_watch.js` fixes it.